config/groups.yaml

```
@@ -59,3 +59,10 @@
     - "2558630"
     - "2562101"
   id: 189659
+  - name: "SSH Tech Folks"
+  members:
+    - "2648650"
+    - "2603823"
+    - "2691579"
+    - "2717380"
+  id: 
```

The syntax isn't right (indentation matters in YAML) and you should omit the `id` field since that will be added automagically.
```suggestion
- name: "SSH Tech Folks"
  members:
    - "2648650"
    - "2603823"
    - "2691579"
    - "2717380"
```